package.json

```diff
@@ -31,7 +31,6 @@
     "node": ">=0.8"
   },
   "scripts": {
-    "pretest": "./node_modules/.bin/jshint --config ./.jshintrc lib test",
```

Comment:
Why was this removed?

Reply:
This script was deprecated and throwing errors where it shouldn't be, so in discussion with @nunofgs, he said that it would be better to remove this and later implement another pre-test method.

Reply:
Should I move this change for another PR?

Reply:
Yes please, I'll merge it quickly :smile:

Reply:
Since the bump done to node engine, this script is working correctly now, so let's keep it for now. 👍